client/landing/stepper/declarative-flow/start-writing.ts

```diff
@@ -70,6 +72,8 @@ const startWriting: Flow = {
 				case 'processing': {
 					// If we just created a new site.
 					if ( ! providedDependencies?.blogLaunched && providedDependencies?.siteSlug ) {
+						console.log( 'SiteIntent.Write', SiteIntent.Write );
+						await setIntent( SiteIntent.Write );
```

Comment:
This doesn't seem to be saving. 🤔

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/76740/commits/e6b680cd4b36dc327b629e1404e122bbbf524b9c